gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -278,7 +273,7 @@ public void showSettings() {
   }
 
   public Observable<ImmutableList<Feature>> getOverlappingFeaturesOnceAndStream() {
-    return overlappingFeaturesSubject;
+    return showFeatureSelectorRequests;
```

Comment:
This now duplicates lines 149-151. We should probably remove `getOverlappingFeaturesOnceAndStream`

Reply:
Done!